tests/framework/extensions/provisioninginput/config.go

```diff
@@ -148,9 +171,28 @@ type Registries struct {
 	RKE2Username   string                       `json:"rke2Username" yaml:"rke2Username"`
 }
 
+type PoolsCommon struct {
```

Comment:
I think this really makes sense, can we remove `common` from the struct name and instead maybe call `Pools`? Common/util naming isn't the best practice in go.

Reply:
sure, I was going off of `RKEClusterSpecCommon` as an example but yes I can remove it